tests/ci/nightly_gpu.yml

```diff
@@ -0,0 +1,108 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+#
+variables:
+  test: 'tests/ci/run_pytest.py'
+  maxnodes : 4
+  reponame : 'Recommenders'
+  branch   : 'azure-pipelines-bz'
```

Comment:
this should be in staging (or master, when it is computed in master), right?

Reply:
Currently, it's only a string that is tagged in an AML experiment for correlation. Eventually, adding code to use the actual pipeline would be useful. 

I changed it so something more generic for now.